### ISSUE TITLE
TypeError: users.filter is not a function

### ISSUE THREAD

**Issue by mboersma** [rauchg/slackin#251]
Our slack-inviter crashed yesterday with this trace:

```
slack-inviter-cmd-751187956-t2jbs -- /srv/www/dist/slack.js:110
slack-inviter-cmd-751187956-t2jbs --       users = users.filter(function (x) {
slack-inviter-cmd-751187956-t2jbs --                     ^
slack-inviter-cmd-751187956-t2jbs -- 
slack-inviter-cmd-751187956-t2jbs -- TypeError: users.filter is not a function
slack-inviter-cmd-751187956-t2jbs --     at SlackData.onres (/srv/www/dist/slack.js:110:21)
slack-inviter-cmd-751187956-t2jbs --     at /srv/www/dist/slack.js:75:16
slack-inviter-cmd-751187956-t2jbs --     at Request.callback (/srv/www/node_modules/superagent/lib/node/index.js:746:30)
slack-inviter-cmd-751187956-t2jbs --     at Request.<anonymous> (/srv/www/node_modules/superagent/lib/node/index.js:135:10)
slack-inviter-cmd-751187956-t2jbs --     at emitOne (events.js:96:13)
slack-inviter-cmd-751187956-t2jbs --     at Request.emit (events.js:188:7)
slack-inviter-cmd-751187956-t2jbs --     at Stream.<anonymous> (/srv/www/node_modules/superagent/lib/node/index.js:938:12)
slack-inviter-cmd-751187956-t2jbs --     at emitNone (events.js:91:20)
slack-inviter-cmd-751187956-t2jbs --     at Stream.emit (events.js:185:7)
slack-inviter-cmd-751187956-t2jbs --     at Unzip.<anonymous> (/srv/www/node_modules/superagent/lib/node/utils.js:124:12)
```

I don't have much more context than that, but wanted to submit the issue in case it points out a code error to someone here.

**Comment by marshall** [comment]
+1 we also saw this in our community running v0.7.3, here's our trace for posterity 
<pre>/usr/lib/node_modules/slackin/node/slack.js:96

        users = users.filter(function (x) {
                      ^
TypeError: undefined is not a function
    at SlackData.onres (/usr/lib/node_modules/slackin/node/slack.js:96:23)
    at /usr/lib/node_modules/slackin/node/slack.js:61:17
    at Request.callback (/usr/lib/node_modules/slackin/node_modules/superagent/lib/node/index.js:746:30)
    at Request.<anonymous> (/usr/lib/node_modules/slackin/node_modules/superagent/lib/node/index.js:135:10)
    at Request.emit (events.js:107:17)
    at Stream.<anonymous> (/usr/lib/node_modules/slackin/node_modules/superagent/lib/node/index.js:938:12)
    at Stream.emit (events.js:129:20)
    at Unzip.<anonymous> (/usr/lib/node_modules/slackin/node_modules/superagent/lib/node/utils.js:124:12)
    at Unzip.emit (events.js:129:20)
    at _stream_readable.js:908:16</pre>

**Comment by zxlin** [comment]
+1 on this issue

```
Tue Jan 24 2017 16:26:42 GMT+0000 (UTC) – online 3, total 285 (+139ms)
Tue Jan 24 2017 16:26:47 GMT+0000 (UTC) – fetching
/usr/lib/node_modules/slackin/dist/slack.js:110
      users = users.filter(function (x) {
                    ^

TypeError: users.filter is not a function
    at SlackData.onres (/usr/lib/node_modules/slackin/dist/slack.js:110:21)
    at /usr/lib/node_modules/slackin/dist/slack.js:75:16
    at Request.callback (/usr/lib/node_modules/slackin/node_modules/superagent/lib/node/index.js:746:30)
    at Request.<anonymous> (/usr/lib/node_modules/slackin/node_modules/superagent/lib/node/index.js:135:10)
    at emitOne (events.js:96:13)
    at Request.emit (events.js:188:7)
    at Stream.<anonymous> (/usr/lib/node_modules/slackin/node_modules/superagent/lib/node/index.js:938:12)
    at emitNone (events.js:91:20)
    at Stream.emit (events.js:185:7)
    at Unzip.<anonymous> (/usr/lib/node_modules/slackin/node_modules/superagent/lib/node/utils.js:124:12)
```

**Comment by zxlin** [comment]
@rauchg I see this has already been fixed by #230 can you release 0.13.1 to npm please?
